src/tests/RequestTest.test.ts

```diff
@@ -0,0 +1,138 @@
+import { request } from 'http';
+import { RequestController } from 'src/api/controllers/RequestController';
+import { Connection } from 'typeorm';
+
+import { UuidParam } from '../api/validators/GenericRequests';
+import { RequestModel } from '../models/RequestModel';
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, DataFactory, RequestFactory, UserFactory } from './data';
+
+let uuidParam: UuidParam;
+let expectedRequest: RequestModel;
+let conn: Connection;
+let requestController: RequestController;
+
+beforeAll(async () => {
```

Comment:
referring back to our discussion in subteam meeting today, lots of this file is probably copied over from another test file with the model renamed to requests. it would be interesting to look into a way to make testing succincter whether that be through a nifty Jest framework or a custom test class where we can pass in the parts of this file that are unique. make an issue for this to address in the future, this can be a refactor for next semester